line-sdk/src/main/java/com/linecorp/linesdk/openchat/OpenChatParameters.java

```diff
@@ -0,0 +1,83 @@
+package com.linecorp.linesdk.openchat;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class OpenChatParameters {
+    private static int DEFAULT_CATEGORY_ID = 17; // Game
+    private OpenChatParameters() {}
+
+    String name;
+    String description;
+    String creatorDisplayName;
+    int categoryId = DEFAULT_CATEGORY_ID;
+    Boolean isSearchable = true;
```

Comment:
If these fields are not supposed to be accessed directly from outside, we need to make it private.

Reply:
addressed in https://github.com/line/line-sdk-android/pull/45/commits/3bb388dc69b8f8096b6816ed1738c9dfd3fc6097